### PR TITLE
Gdetect

### PR DESCRIPTION
**Pull request by tdidailler**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR adds a Pack to use GLIMPS GDetect in it.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by edik24** [comment]
Docker PR is here:
https://github.com/demisto/dockerfiles/pull/7650

File: Packs/GLIMPS_Detect/Integrations/GLIMPSDetect/GLIMPSDetect.yml [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+  type: 0
+- additionalinfo: Maximum number of alerts per fetch. Default and recommended is 50
+  defaultvalue: '50'
+  display: Fetch Limit
+  name: max_fetch
+  required: false
+  type: 0
+script:
+  feed: false
+  isfetch: true
```
Comment (MLainer1) [inline Packs/GLIMPS_Detect/Integrations/GLIMPSDetect/GLIMPSDetect.yml:52]:
```suggestion
  isfetch: false
```
There is no `fetch incidents` command. Do you intend on adding it? if not, please change

File: Packs/GLIMPS_Detect/Integrations/GLIMPSDetect/GLIMPSDetect.yml [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+  display: First Fetch Time
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: Maximum number of alerts per fetch. Default and recommended is 50
+  defaultvalue: '50'
+  display: Fetch Limit
+  name: max_fetch
+  required: false
+  type: 0
```
Comment (MLainer1) [inline Packs/GLIMPS_Detect/Integrations/GLIMPSDetect/GLIMPSDetect.yml:49]:
```suggestion
```
There is no fetch incidents command. Do you intend on adding it? if not, please change

**Comment by MLainer1** [comment]
@tdidailler Hi, make sure to look at the validation failures, and sign the [contributions form](https://forms.gle/XDfxU4E61ZwEESSMA)

**Review by MLainer1 — APPROVED** [review]

**Pull request merged**